src/rnp/rnpcfg.c

```diff
@@ -346,6 +346,7 @@ rnp_cfg_get_pswdtries(rnp_cfg_t *cfg)
     }
 }
 
+#if 0
```

Comment:
I would prefer we remove this if it's not used. we can still get it from github history if needed

Reply:
Yeah probably better, if everyone agrees we don't need them right now / in immediate future.